hledger-lib/hledger-lib.cabal

```diff
@@ -110,6 +110,7 @@ library
       Hledger.Data.Dates
       Hledger.Data.Journal
       Hledger.Data.Ledger
+      Hledger.Data.MarketPrice
```

Comment:
new show function is indeed not yet used within `hledger` itself.  But I am using it directly in some of my own utility scripts (https://github.com/mstksg/jle-utils/blob/master/app/UpdatePrices.hs#L75).  Looking into being able to have parse+print being a non-lossy process is a good next step that this show function could hopefully be used for!